CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ set(Boost_USE_MULTITHREADED ON)
 set(Boost_USE_STATIC_RUNTIME OFF)
 find_package(
         Boost
-        1.58 # Minimum version, vetted via Ubuntu 16.04
+        1.68
```

Comment:
So this is actually a minimum version number, not an explicit version. FindBoost is wonky. Did you need the upgrade?

Reply:
Nah, didn't run into any problems. Thought I'd change this number so people can use 1.68 features.